src/client/src/containers/PostGenerationModal/index.tsx

```diff
@@ -112,57 +115,171 @@ const PostGenerationModal = ({
     if (isTemplatesFailed) {
```

Comment:
We should remove `handleCloseWizard` from line 90 now (we can clean up everything in a separate PR if u want including constants, etc)

Reply:
good catch, function removed.